### ISSUE TITLE
Remove padding-rigth on Message Component

### ISSUE THREAD

**Issue by eduguillem-sage** [Sage/carbon#5904]
### Desired behaviour

When using a [Message](https://carbon.sage.com/?path=/docs/message--default-story) component and close button is not shown, padding right should be equals than left.

### Current behaviour

When using a [Message](https://carbon.sage.com/?path=/docs/message--default-story) component and close button is not shown, this place is not fulled with text. Padding-right is maintained.
![Captura de pantalla 2023-03-08 a las 12 37 28](https://user-images.githubusercontent.com/111743200/223728156-bc189c81-f090-47a9-99b0-f2af18facbd9.png)

### Suggested Solution

Maybe close button should not be position absolute.

### CodeSandbox or Storybook URL

https://codesandbox.io/s/inspiring-roentgen-zyxj6z

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by ljemmo** [comment]
@nineteen88 this is correct. When no close is shown, the right-padding should be the same as the left padding. {spacing.300}
